requirements.txt

```diff
@@ -54,15 +55,15 @@ fs==0.5.4
 ftfy==4.4.3
 ftputil==3.3.1
 funcsigs==1.0.2
-functools32==3.2.3.post2
+functools32===3.2.3.post2;python_version=='2.7'
 furl==0.5.6
 future==0.18.2
 html5lib==1.1
 hyperlink==19.0.0
 idna==2.7
 IDUtils==1.1.7
 imagesize==1.2.0
-importlib-metadata==0.18
+importlib-metadata==1.0.0
```

Comment:
do we need these changes?

Reply:
mm... Maybe not, because it was raising an error when we tried to run it without docker container